features/promotion/applying_catalog_promotions/applying_all_available_types_catalog_promotions.feature

```diff
@@ -0,0 +1,35 @@
+@applying_catalog_promotions
+Feature: Applying all available catalog promotions
```

Comment:
IMHO this scenario should be written from Shop Owner Perspective

Reply:
I'm not 100% sure 😄 The clear beneficient of working catalog promotions is Customer/Visitor, as they receive a lower price for the product they buy. It's also then consistent with `Applying catalog promotions for variants` feature

Reply:
If I see correctly most of scenarios are written from a customer perspective, so some should be written from a Shop Owner perspective too.

Reply:
But the argument of proportions is not a good reason to change the beneficent 😄 The question is: who gets a benefit from the proposed feature. IMO it's Customer/Visitor - and the fact that they're are also beneficent in other feature that are very similar is a reason to put them also here 🖖